frontend/src/general/userForm/__tests__/FormConsole.test.jsx

```diff
@@ -0,0 +1,160 @@
+import {
+  afterEach,
+  beforeEach,
+  describe,
+  expect,
+  it,
+  jest
+} from "@jest/globals";
+import React from "react";
+import Enzyme from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import { Button, Container, TextField } from "@material-ui/core";
+import FormConsole from "../FormConsole";
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe("<FormConsole/> class component", () => {
+  let wrapper;
+  const mockHandleSubmit = jest.fn();
+  const dummyValues = {
+    handleSubmit: mockHandleSubmit,
+    buttonName: "Create"
+  };
+
+  beforeEach(() => {
+    wrapper = Enzyme.shallow(
+      <FormConsole
+        handleSubmit={dummyValues.handleSubmit}
+        buttonName={dummyValues.buttonName}
+        isValidate
+        isCreateUser
+      />
+    );
+  });
+
+  afterEach(() => {
+    wrapper.unmount();
+  });
+
+  describe("render()", () => {
+    it("should have 1 <Container/>, 2 <TextFields/> and 1 <Button/>", () => {
+      expect(wrapper.find(Container)).toHaveLength(1);
+      expect(wrapper.find(TextField)).toHaveLength(2);
+      expect(wrapper.find(Button)).toHaveLength(1);
+    });
+
+    it("contains the props buttonName as a child of <Button/>", () => {
+      expect(wrapper.find(Button).children().text()).toEqual(
+        dummyValues.buttonName
+      );
+    });
+
+    it("if isValidate prop is true, it should have the form's noValidate to true", () => {
```

Comment:
❗ test name. "it should have the form's noValidate to be **false**" based on your expect below

Reply:
Good catch!